cmd/api-linter/cli.go

```diff
@@ -305,3 +295,73 @@ func getOutputFormatFunc(formatType string) formatFunc {
 	}
 	return yaml.Marshal
 }
+
+func resolveImports(protoImportPaths []string) []string {
```

Comment:
Nit: maybe simplify the name `protoImportPaths` to `imports`  https://go.dev/wiki/CodeReviewComments#variable-names